### PR TITLE
Prevent constant updating in nearby view

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**

Fixes constant rerendering of the nearby view by only updating `currentPosition` if the user moves .0001 lat/long away from their initial position. 

Is this the most efficient way to do this? Should this instead be set in the nearby view? Should we potentially calc the km distance between the two points instead, to make it more geographically consistent? Open to a lot of feedback on this 

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/viewers/nearby/nearby-view.tsx [hunk 1]
```diff
@@ -246,6 +247,8 @@ function NearbyView({
       </li>
     ))
 
+  console.log(nearbyItemList)
```
Comment (daniel-heppner-ibigroup) [inline lib/components/viewers/nearby/nearby-view.tsx]: remove this!

**Review by daniel-heppner-ibigroup — CHANGES_REQUESTED** [review]
- Update snapshots
- Fix percy tests
- Maybe add a comment that talks about approximately what the distance of .001 lat/lon is. I think it depends on location

Maybe we should just not update the position based on currentPosition changes?

File: lib/components/viewers/nearby/nearby-view.tsx (hunk 1 above)
Comment (amy-corson-ibigroup) [reply lib/components/viewers/nearby/nearby-view.tsx]: Oops!

**Comment by amy-corson-ibigroup** [comment]
Changes to the percy snapshots are being caused by changes to the `h3`! Before the invisible header was taking up space, now it's not

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Thanks for the fixes!

File: lib/components/app/responsive-webapp.js [hunk 2]
```diff
@@ -119,6 +138,25 @@ class ResponsiveWebapp extends Component {
       }
     }
 
+    // Watch for position changing on mobile
+    if (isMobile()) {
+      navigator.geolocation.watchPosition(
+        // On success
+        (position) => {
+          const shouldUpdate = this.positionShouldUpdate(position)
+          if (shouldUpdate) {
```
Comment (miles-grant-ibigroup) [inline lib/components/app/responsive-webapp.js:147]: You can inline this. It's only used once

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks and works great! Wonderful work. Just one minor nitpick

**Pull request merged**